src/drivers/mpu9250/mpu9250.cpp

```diff
@@ -328,7 +328,9 @@ MPU9250::init()
 #endif
 
 	/* do CDev init for the mag device node, keep it optional */
```

Comment:
Potentially, but not at the moment. Only once we start adding bus options to https://github.com/PX4/Firmware/blob/master/src/drivers/mpu9250/main.cpp#L113 with equal `busid`'s.